docs/content/tutorial/step_06.ngdoc

```diff
@@ -230,6 +230,9 @@ You can now rerun `npm run protractor` to see the tests run.
 
 * Reverse the sort order by adding a `-` symbol before the sorting value:
   `<option value="-age">Oldest</option>`
+  Upon refreshing the page, you'll see that the default selection of the "age" option in "Sort by" is blank again.
+  Try fixing this in **`app/phone-list/phone-list.component.js`:**
```

Comment:
There is a redundand `:` after the filename and the `.` is missing at the end of the sentence.
I would also change wrap the filename in `` ` `` instead of `**` for consistency.

Reply:
I've deleted the redundant `:` and also added the `.` at the end of the sentence. Fixed the filename wrap as well. Fixed in https://github.com/angular/angular.js/pull/16781/commits/41e28b65e4ad8d18fef7a74ff57d3944c4caa4c4